site/content/blog/release-0.30.0.md

```diff
@@ -0,0 +1,160 @@
+---
+title: "Release 0.30.0 is out!"
+date: 2023-02-24T09:50:29+11:00
+draft: true
+---
+
+`rust-bitcoin` version 0.30.0 is out now.
+
+<!--more-->
+
+This is a rather large release so we decided to write an update guide for you guys. If this guide is
+not useful or lacking in some way please do let us know so we can do better.
+
+First a little excuse for why this is going to be so painful. We have, and do, try to deprecate
+things if we make API breaking changes. We do this using
+
+     #[deprecated(since = "x.y.z", note = "use foobar instead")]
+
+This allows us to give you a hint on how to upgrade by way of the compiler. The problem we hit was
+that its not always possible to deprecate things (e.g. changing arguments to a function) so under
+the cover of "this is a pre-1.0 release" and with the aim of pushing kind of fast so we can get to
+the 1.0 release, we got a bit sloppy with deprecation this release - sorry about that. We are very
+much trying to get to a place where we can commit to our APIs and stabilize the codebase, that is
+the primary goal of development at the moment. If you have API changing suggestions or requests
+please get them into us now so your needs can be met.
+
+Without further ado, here is the upgrade guide. Enjoy!
+
+## Suggested steps
+
+We suggest that you take these steps when upgrading:
+
+0. Make sure to update other dependency versions in `Cargo.toml` if you use them explicitly: `bitcoin_hashes` to 0.12.0, `secp256k1` to 0.26.0
+1. Remove all occurrences of `util::` referring to our crate
+2. Replace `Script` with `ScriptBuf` (`s/\([^A-Za-z0-9]\)Script\([^A-Za-z0-9]\)/\1ScriptBuf\2/g` should work in most cases)
+3. Replace instances of `.parse::<Address>()` with `.parse::<Address<_>>()`
+4. Call `require_network(network)` on parsed addresses (you'll get no method found for `Address<NetworkUnchecked>` errors)
+5. Replace `locktime` with `locktime::absolute`
+6. Replace `PackedLockTime` with just `LockTime`
+7. Import key types from the `key` submodule rather than `schnorr` or `edsa`
```

Comment:
In c69e3e8662180695ed9d0c32910c340777d62098:

nit: `ecdsa`

Reply:
Woops, thanks.